src/devices/Hmc5883l/Hmc5883l.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Numerics;
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// 3-Axis Digital Compass HMC5883L
+    /// </summary>
+    public class Hmc5883l : IDisposable
+    {
+        /// <summary>
+        /// HMC5883L I2C Address
+        /// </summary>
+        public const byte I2cAddress = 0x1E;
```

Comment:
I think this should be private

Reply:
The sensor address need to be made public so that I2cConnectionSetting can use it

Reply:
Agreed, this should stay private. Does this chip allow an alternate I2cAddress? If so, we should expose that as well.

Reply:
There is only one I2C address, and if there are multiple addresses, I will use enum type as ADS1115 does.